modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -600,8 +600,8 @@ object ElasticRequest {
     def size(value: Int): SearchRequest =
       self.copy(size = Some(value))
 
-    def sort(sorts: Sort*): SearchRequest =
-      self.copy(sortBy = sortBy ++ sorts.toSet)
+    def sort(sort: Sort, sorts: Sort*): SearchRequest =
+      self.copy(sortBy = (sortBy :+ sort) ++ sorts.toList)
```

Comment:
If we change `(sortBy :+ sort) ++ sorts.toList` to `sortBy ++ (sort :: sorts.toList)` then `sortBy` only gets copied once instead of twice.
Also, should we use Chunk instead?